wilkes/roster.json

```diff
@@ -52,7 +52,12 @@
     },
     {
       "name": "Immanuel Egboche",
-      "github": "ImmanuelEgboche"
+      "github": "ImmanuelEgboche",
+      "materials": [
+        {
+          "type": "cv"
+      }
+      ]
```

Comment:
Can you please indent your code (align the curly braces and the square brackets) :)?